terra/logger.py

```diff
@@ -71,9 +71,13 @@
 import struct
 import select
 import pickle
+import functools
```

Comment:
Is `functools` used?

Reply:
Nope, flask8 caught that (and a few others) too ;)